scripts/items/loot/rf_geist_tear_item.nut

```diff
@@ -0,0 +1,77 @@
+this.rf_geist_tear_item <- this.inherit("scripts/items/item", {
+	m = {},
+	function create()
+	{
+		this.item.create();
+		this.m.ID = "misc.rf_geist_tear";
+		this.m.Name = "Geist Tear";
+		this.m.Description = "When a geist leaves this plane, sometimes a gem is left behind. Wise men believe it is the ethereal essence of the spirit made manifest.";
+		this.m.Icon = "loot/rf_geist_tear.png";
+		this.m.SlotType = ::Const.ItemSlot.None;
+		this.m.ItemType = ::Const.Items.ItemType.Misc | ::Const.Items.ItemType.Loot;
+		this.m.IsDroppedAsLoot = true;
+		this.m.Value = 300;
+	}
+
+	function getTooltip()
+	{
+		local result = [
+			{
+				id = 1,
+				type = "title",
+				text = this.getName()
+			},
+			{
+				id = 2,
+				type = "description",
+				text = this.getDescription()
+			}
+		];
+		result.push({
+			id = 66,
+			type = "text",
+			text = this.getValueString()
+		});
+		return result;
+	}
+
+	function getBuyPrice()
+	{
+		if (this.m.IsSold)
+		{
+			return this.getSellPrice();
+		}
+
+		if (("State" in ::World) && ::World.State != null && ::World.State.getCurrentTown() != null)
+		{
+			return ::Math.max(this.getSellPrice(), ::Math.ceil(this.getValue() * 1.5 * ::World.State.getCurrentTown().getBuyPriceMult() * ::World.State.getCurrentTown().getBeastPartsPriceMult()));
+		}
+		else
+		{
+			return ::Math.ceil(this.getValue());
+		}
+	}
+
+	function getSellPrice()
+	{
+		if (this.m.IsBought)
+		{
+			return this.getBuyPrice();
+		}
+
+		if (("State" in ::World) && ::World.State != null && ::World.State.getCurrentTown() != null)
+		{
+			return ::Math.floor(this.getValue() * ::Const.World.Assets.BaseLootSellPrice * ::World.State.getCurrentTown().getSellPriceMult() * ::World.State.getCurrentTown().getBeastPartsPriceMult() * ::Const.Difficulty.SellPriceMult[::World.Assets.getEconomicDifficulty()]);
+		}
+		else
+		{
+			return ::Math.floor(this.getValue());
+		}
+	}
+
+	function playInventorySound( _eventType )
+	{
+		this.Sound.play("sounds/combat/armor_leather_impact_03.wav", ::Const.Sound.Volume.Inventory);
```

Comment:
I think the glass bottle sound would be better i.e. the one for holy water bottle or potions etc. Or some other sound that goes more in line with it being a gem/glass? Not a huge deal though.

Reply:
I used the same as the Soul Splinter, which is where the recolored icon comes from. I would like to leave it as is for now.